ebuild-writing/functions/pkg_config/text.xml

```diff
@@ -49,7 +48,7 @@ Taken from the <c>mysql</c> ebuilds. Note the use of <c>${ROOT}</c>.
 
 <codesample lang="ebuild">
 pkg_config() {
-	if [ ! -d "${ROOT}"/var/lib/mysql/mysql ] ; then
+	if [[ ! -d "${ROOT}"/var/lib/mysql/mysql ]] ; then
```

Comment:
No quotation marks here.

Reply:
Really? This is common style.

Reply:
Which doesn't make it right. `[[ ]]` is a syntax element, and no word splitting or globbing will take place. So the quotation marks are excessive (but they were necessary inside `[ ]`).

Reply:
You're right, of course, but I've noticed a common theme between both of the changes I've suggested here.

Both are imperfect (and one is unnecessary) but both _suggest_ better habits, even if imperfect. I'm happy to drop it here though.

Reply:
Yes, please drop the quotes. If you leave them in place, there's little point in using `[[ ]]` instead of `[ ]`.

Of course this is very minor, so I wouldn't file a bug for seeing quotes in `[[ ]]`. But I would comment about it in eclass review.

Reply:
Understood. Let's fix that.